### DISCUSSION TITLE
Saved profile export/import for another pc?

### DISCUSSION THREAD

**Discussion by nbfarhad** [Eugeny/tabby#5669] [Q&A]
I couldn't find any profile export option for all my ssh connectiosns, I want to export all my saved profiles from linux tabby to Mac or Windows tabby.
Is there any option to do that?

**Comment by Eugeny** [comment]
There is no export/import function yet, but you can copy over the profiles from the config file (Settings -> Config file).

**Reply by Eugeny** [reply]
The IDs are just randomly generated.

**Reply by pratik149** [reply]
Thanks much! This did the trick for my team, but a dedicated import/export would definitely help.

**Reply by Andrew0x2** [reply]
I had a list of more than 300 IPs to add ssh protocol to Tabby, this script i created did the job.
`#! /bin/bash

while IFS="|" read -r id desc ip
do
    echo "  - type: ssh"
    echo "    name: $id - $desc"
    echo "    icon: fas fa-desktop"
    echo "    options:"
    echo "      host: $ip"
    echo "      user:  "  #ADD USER
    echo "      algorithms: {}"
    echo "      auth: password"
    echo "    weight: -1"
    echo "    group: " #ADD GROUP IF YOU WANT
    echo "    id: ssh:custom:$id$desc:$(cat /proc/sys/kernel/random/uuid)"
	
done < <(tail -n +1 file.txt)   #<----- FILE 
` 
In file.txt file you must enter the ID, Name and IP separated by the | character.
like this : 
ID0|NAME0|IP
ID1|NAME1|IP

With the output you can generate the conf.

**Reply by locksec** [reply]
I see this is a few years old now, but I would love a way to copy my profiles to another machine. Are they not stored on disk somewhere?

**Reply by mrmango** [reply]
For info, go settings>config file-> Show Config File (botton right of screen).

Copy this file (I guess would copy settings over)

**Reply by ohgadd** [reply]
> I see this is a few years old now, but I would love a way to copy my profiles to another machine. Are they not stored on disk somewhere?

%APPDATA%\tabby\config.yaml
